repository/service_request_repository.go

```diff
@@ -0,0 +1,35 @@
+package repository
```

Comment:
you can name the file as just `service_request.go`. the word `repository` is already part of the package name.